_guidelines-v4/01-introduction/02-basicConcepts/04-meicustomization.md

```diff
@@ -0,0 +1,13 @@
+---
+version: "v4"
+title: "Customizing MEI"
+sectionid: "meicustomization"
+---
+
+In production, it is best to use a customized version of MEI, restricted to the very needs of a project. Such a custom schema will guide the encoders and will help to ensure consistency and data quality throughout a project's files. A customization typically provides a subset of MEI's encoding models, with only one solution for any given situation being allowed. The customization will help to reflect the scope of a project into its data: Only those aspects of music notation a project is interested in will be allowed, so that the absence of a specific information can not be misunderstood as an oversight of the encoders. Larger editorial projects like _Complete Works_ editions typically use _Editorial Guidelines_ (german: _Editionsrichtlinien_) for the same purposes: (internal) quality control and (external) documentation. In that sense, MEI customizations may serve as Editorial Guidelines in digital form.
```

Comment:
Please clarify a little more about the connection of these customizations to the aforementioned MEI profiles. I.e. "customized version of MEI" means actually "even more specific, customized derivation based on a pre-built MEI _profile_"?